src/components/Switch/Switch.component.tsx

```diff
@@ -29,7 +30,7 @@ export const Switch = (props: SwitchProps) => {
             id={id}
             fullWidth={fullWidth}
         >
-            <Form.Check.Input
+            {/* <Form.Check.Input
                 className='mm-switch_input'
```

Comment:
I guess this piece of code was commented out by @sooraj-shukla to prevent the storybook from breaking due to bootstrap version downgrade before we started fixing the components. This change is visible here because the base of this PR is changed now due to merging of PRs